packages/cli/lib/sandbox-sync.js

```diff
@@ -0,0 +1,173 @@
+const Spinnies = require('spinnies');
+const { getHubSpotWebsiteOrigin } = require('@hubspot/cli-lib/lib/urls');
+const { logger } = require('@hubspot/cli-lib/logger');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const {
+  getAvailableSyncTypes,
+  pollSyncTaskStatus,
+  getAccountName,
+} = require('./sandboxes');
+const { initiateSync } = require('@hubspot/cli-lib/sandboxes');
+const { logErrorInstance } = require('@hubspot/cli-lib/errorHandlers');
+const {
+  isSpecifiedError,
+  isMissingScopeError,
+} = require('@hubspot/cli-lib/errorHandlers/apiErrors');
+const { getSandboxTypeAsString } = require('./sandboxes');
+
+const i18nKey = 'cli.commands.sandbox.subcommands.sync';
+
+/**
+ * @param {Object} accountConfig - Account config of sandbox portal
+ * @param {Object} parentAccountConfig - Account config of parent portal
+ * @param {String} env - Environment (QA/Prod)
+ * @param {Boolean} allowEarlyTermination - Option to allow a keypress to terminate early
+ * @returns
+ */
+const syncSandbox = async ({
+  accountConfig,
+  parentAccountConfig,
+  env,
+  allowEarlyTermination = true,
```

Comment:
Separated sandbox sync logic into this util. We call this during the creation flow for standard sandboxes